### PR TITLE
Jmeter script for postgresql query load generation

### PR DESCRIPTION
**Pull request by oumkale**
**What this PR does / why we need it**:

Jmeter script for postgresql query load generation

Logs:
```
OpenJDK 64-Bit Server VM warning: If the number of processors is expected to increase from one, then you should configure the number of parallel GC threads appropriately using -XX:ParallelGCThreads=N
Aug 31, 2021 9:59:53 AM java.util.prefs.FileSystemPreferences$1 run
INFO: Created user preferences directory.
Creating summariser <summary>
Created the tree successfully using postgres-load.jmx
Starting standalone test @ Tue Aug 31 09:59:53 UTC 2021 (1630403993676)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
summary +    345 in 00:00:05 =   66.3/s Avg:    60 Min:    10 Max:  1164 Err:     0 (0.00%) Active: 5 Started: 5 Finished: 0
summary +   2714 in 00:00:30 =   90.6/s Avg:    55 Min:     9 Max:   314 Err:     0 (0.00%) Active: 5 Started: 5 Finished: 0
summary =   3059 in 00:00:35 =   87.0/s Avg:    55 Min:     9 Max:  1164 Err:     0 (0.00%)
summary +   2962 in 00:00:30 =   98.7/s Avg:    50 Min:    10 Max:   175 Err:     0 (0.00%) Active: 5 Started: 5 Finished: 0
summary =   6021 in 00:01:05 =   92.4/s Avg:    53 Min:     9 Max:  1164 Err:     0 (0.00%)
summary +   2742 in 00:00:30 =   91.4/s Avg:    54 Min:    10 Max:   209 Err:     0 (0.00%) Active: 5 Started: 5 Finished: 0
```

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: custom/workflow-helper/postgres-helper/jmeter/deploy-jmeter.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      app: postgres
+  template:
+    metadata:
+      labels:
+        app: postgres
+    spec:
+      serviceAccountName: postgres-operator
+      containers:
+      - name: postgres
+        image: "litmuschaos/litmus-pg-jmeter:master"
```
Comment (ispeakc0de) [inline custom/workflow-helper/postgres-helper/jmeter/deploy-jmeter.yaml:21]:
```suggestion
        image: "litmuschaos/litmus-pg-jmeter:latest
```

**Review by ispeakc0de — DISMISSED** [review]

File: Makefile [hunk 2]
```diff
@@ -285,6 +285,16 @@ _push_litmus_git_app_checker:
 
 litmus-git-app-checker: deps _build_litmus_git_app_checker _push_litmus_git_app_checker
 
+_build_litmus_pg_jmeter:
```
Comment (uditgaurav) [inline Makefile:288]: Should we also add it in `.github/workflows` ?

Comment (oumkale) [reply Makefile:288]: Sure @uditgaurav

File: .github/workflows/push.yml [hunk 3]
```diff
@@ -480,3 +483,14 @@ jobs:
...
+  litmus-pg-jmeter:
+    runs-on: ubuntu-latest
+    needs: changes
+    if: needs.changes.outputs.litmus-pg-jmeter == 'true'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Building litmus-pg-jmeter image
+        run: make litmus-pg-jmeter
```
Comment (uditgaurav) [inline .github/workflows/push.yml:496]: Can we directly use `make _push_litmus_pg_jmeter` here, as it'll do both build and push multiarch.

Comment (oumkale) [reply .github/workflows/push.yml]: Thank you @uditgaurav, Updated PTAL

File: custom/workflow-helper/postgres-helper/jmeter/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,27 @@
+FROM centos:7.5.1804
```
Comment (uditgaurav) [inline custom/workflow-helper/postgres-helper/jmeter/Dockerfile:1]: The base image [centos:7.5.1804](https://hub.docker.com/layers/centos/library/centos/centos7.5.1804/images/sha256-cf49811e3cdb94cbdfd645f3888d7add06a315449cf2c7ca7b81c312f1e46c63?context=explore) does not support multiarch platform it is only built for amd64 looks like so it should not work on arm64 I think. Can we check with any other centos that supports both platforms? like [centos7](https://hub.docker.com/layers/centos/library/centos/centos7/images/sha256-b98bfc4a7094a98ed52d49f9558bb15a4cee9854f182d30085d459d4a55d2e03?context=explore), [centos7.9.2009
](https://hub.docker.com/layers/centos/library/centos/centos7.9.2009/images/sha256-1cef7b00a46a2a90c62eef9a9707d01d19c2f5e253b7ccc574ccaa775063d7e0?context=explore) or [any centos 8](https://hub.docker.com/_/centos?tab=tags&page=1&ordering=last_updated&name=8)

Comment (oumkale) [reply custom/workflow-helper/postgres-helper/jmeter/Dockerfile:1]: have updated centos8

**Review by uditgaurav — APPROVED** [review]

File: .github/workflows/push.yml [hunk 5]
```diff
@@ -480,3 +483,14 @@ jobs:
...
+  litmus-pg-jmeter:
+    runs-on: ubuntu-latest
+    needs: changes
+    if: needs.changes.outputs.litmus-pg-jmeter == 'true'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Building litmus-pg-jmeter image
+        run: make _push_litmus_pg_jmeter
```
Comment (uditgaurav) [inline .github/workflows/push.yml:496]: you can also add [make docker.buildx](https://github.com/litmuschaos/test-tools/blob/7d8559fe8effb62c1c87aef8050a7c7df62f90e2/.github/workflows/push.yml#L188) before calling push

**Review by uditgaurav — APPROVED** [review]

**Pull request merged**